code/controllers/subsystem/ticker.dm

```diff
@@ -228,6 +228,8 @@ var/datum/subsystem/ticker/ticker
 			if(M.stat != DEAD)
 				var/turf/T = get_turf(M)
 				if(T && T.z==1)
+					if(corebunker && istype(get_area(M), /area/turret_protected/ai))
```

Comment:
Ehh. having the bunker be wherever the AI happens to be is kinda weird tbh. Although yeah, The issue with the area remaining if the core is eaten is a problem.